easybuild/easyconfigs/p/Python/Python-2.7.9-gompi-1.5.16.eb

```diff
@@ -0,0 +1,24 @@
+name = 'Python'
+version = '2.7.9'
+
```

Comment:
it is common, yes, we have several `-bare` Python easyconfigs included in the last EasyBuild releases

see https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/p/Python/